### PR TITLE
Skip patches via gitignore

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -52,3 +52,6 @@ _output/
 
 # bin 
 bin/
+
+# patch
+*.patch
```
Comment (msau42) [inline .gitignore]: @pohly brought this up in the kubernetes-csi standup meeting.  We think actually the .gitignore should only include files/binaries that are generated by this repository and not for files dependent on a dev's environment.

So for this particular repo, we should just ignore bin/

Comment (davidz627) [reply .gitignore]: for personal dev environments its possible to set up a local git ignore, this page has a guide for it: https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b

**Comment by humblec** [comment]
@msau42 closing this PR based on comments from @msau42 .

**Pull request closed without merging**